### DISCUSSION TITLE
UnderlineNav/banner/gutter on profile is too tall

### DISCUSSION THREAD

**Discussion by rossjrw** [community/community#22194] [Accessibility]
<p>The height of the dark theme toggle creates excess vertical space in the navigation banner at the top of users’ profile pages:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/7/8755b928f4d2b3ccf841b6fa3e7047b7412045a6.png" data-download-href="/uploads/short-url/jje7qu9EAUMML9kJmcDcqc8AFj8.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29130152/181069425-43d6d8db-07fa-4701-89d8-1c544e8351a1.png" alt="image" data-base62-sha1="jje7qu9EAUMML9kJmcDcqc8AFj8" width="690" height="75" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/7/8755b928f4d2b3ccf841b6fa3e7047b7412045a6_2_690x75.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/7/8755b928f4d2b3ccf841b6fa3e7047b7412045a6_2_1035x112.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/7/8755b928f4d2b3ccf841b6fa3e7047b7412045a6_2_1380x150.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/7/8755b928f4d2b3ccf841b6fa3e7047b7412045a6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1917×211 27.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The problem is more obvious scrolled down the page a bit, where the account name is misaligned with the navigation links as a result of the height change:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/b/5b2268ec0f01c027fcd902c3d573dbd72cde069e.png" data-download-href="/uploads/short-url/d0daHokS4JtnwYyz3gsuhzdJZuu.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29130152/181069427-8b7e2866-0c31-44b3-a7b9-99d9a02adc87.png" alt="image" data-base62-sha1="d0daHokS4JtnwYyz3gsuhzdJZuu" width="690" height="45" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5b2268ec0f01c027fcd902c3d573dbd72cde069e_2_690x45.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5b2268ec0f01c027fcd902c3d573dbd72cde069e_2_1035x67.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5b2268ec0f01c027fcd902c3d573dbd72cde069e_2_1380x90.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5b2268ec0f01c027fcd902c3d573dbd72cde069e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1619×106 12.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Compare this to a screenshot at the same width, except with the toggle removed and the <code>hx_UnderlineNav-with-profile-color-modes-banner</code> class omitted. The banner height is much more appropriate for the content:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/8/98654e476e52478bc28ad49dc3367b6b2e4b900d.png" data-download-href="/uploads/short-url/lK9CzuicL8CLLzIAn5kn5QHh5aB.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29130152/181069428-9b680db2-89ba-4767-9083-93791c63a17b.png" alt="image" data-base62-sha1="lK9CzuicL8CLLzIAn5kn5QHh5aB" width="690" height="29" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/8/98654e476e52478bc28ad49dc3367b6b2e4b900d_2_690x29.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/8/98654e476e52478bc28ad49dc3367b6b2e4b900d_2_1035x43.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/8/98654e476e52478bc28ad49dc3367b6b2e4b900d_2_1380x58.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/8/98654e476e52478bc28ad49dc3367b6b2e4b900d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1620×70 8.55 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This could possibly be resolved with a max-height set on the toggle image, or a negative vertical margin.</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for the attention to detail and for sharing this issue with us <a class="mention" href="/rossjrw">@rossjrw</a>! The dark mode toggle is actually a temporary promo banner for dark mode at Universe but since it will be sticking around for a bit longer I’ve added this to our backlog to review</p>

**Comment by cehfisher** [comment]
Hi @rossjrw - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now as the toggle was a temporary feature. Could you please confirm and close this issue? Thank you! (cc @queenofcorgis)
